app/Widgets/Literacy/README.md

```diff
@@ -0,0 +1,48 @@
+# Literacy for Digital Atlas
+
+This Literacy was designed for the Digital Atlas.  It ...
```

Comment:
Can you add a description for the widget here.  Also provide attribution for where you retrieved the data.  You can use a Markdown link to link to the data file.

Reply:
@LeviCostello Great job on the updates!  Sorry, there is one final issue.  Can you add a brief description of the widget in the README file.  Thanks!